tests/rt.conf

```diff
@@ -396,6 +402,24 @@ COMPILE | CCPP=Y STATIC=Y SUITES=FV3_GSD_v0,FV3_GFS_v15_thompson 32BIT=Y DEBUG=Y
 RUN     | fv3_ccpp_gsd_debug                                                                                                             | standard    |                | fv3         |
 RUN     | fv3_ccpp_thompson_debug                                                                                                        | standard    |                | fv3         |
 RUN     | fv3_ccpp_thompson_no_aero_debug                                                                                                | standard    |                | fv3         |
+#######################################################################################################################################################################################
```

Comment:
Future work: when we finally drop the IPD tests, we need to clean up and reorganize the remaining tests.